File: math/105torus/105torus_2019/src/my_error.c
/*
** EPITECH PROJECT, 2020
** my_error
** File description:
** 105
*/

#include "my.h"
#include "105torus.h"

void my_error(char const *str)
{
    int output_error = 2;

    write(output_error, str, strlen(str));
    exit(84);
}

File: math/110borwein/110borwein_2019/src/my_error.py
##
## EPITECH PROJECT, 2020
## my_error.py
## File description:
## my_error.py
##

import sys
from src import my_usage

def my_error(argv):
    if len(argv) < 2:
        sys.stderr.write("Error: Too few arguments\n")
        exit(84)
    elif len(argv) > 2:
        sys.stderr.write("Error: Too many arguments\n")
        exit(84)
    if argv[1] == "-h":
        my_usage.my_usage()
    if argv[1].isnumeric() == False:
        sys.stderr.write("Error: All arguments must be positive numbers\n")
        exit(84)